src/utils/narrow.js

```diff
@@ -537,3 +537,24 @@ export const getNarrowForReply = (message: Message | Outbox, ownUserId: UserId)
     return topicNarrow(streamName, message.subject);
   }
 };
+
+/**
+ * Answers, this question, "What should be the right accessibilityLabel for
+ * send message button?"
```

Comment:
So, the reason I suggested putting it in narrow.js was that I figured it could end up being useful for things besides the send message button; my thought was that it could be broadly useful for a lot of different areas in the UI. So this comment is too specific. 🙂 Possibly most of its uses will be for improving our interface with screen readers.

How about something like, "A translated description of a narrow, useful for screen-reader interactions."

Then, in that case, please do put useful descriptions for all types of narrows (`allPrivate`, `home`, etc.).